Add audio mediaType (#6183)
prebid/prebid.github.io: dev-docs/adunit-reference.md

| [`banner`](#adUnit.mediaTypes.banner) | At least one of the `banner`, `native`, or `video` objects are required. | Object | Defines properties of a banner ad. For examples, see [`adUnit.mediaTypes.banner`](#adUnit.mediaTypes.banner). |
| [`native`](#adUnit.mediaTypes.native) | At least one of the `banner`, `native`, or `video` objects are required. | Object | Defines properties of a native ad. For properties, see [`adUnit.mediaTypes.native`](#adUnit.mediaTypes.native). |
| [`video`](#adUnit.mediaTypes.video) | At least one of the `banner`, `native`, or `video` objects are required. | Object | Defines properties of a video ad. For examples, see [`adUnit.mediaTypes.video`](#adUnit.mediaTypes.video). |

> | [`audio`](#adUnit.mediaTypes.audio) | At least one of the `banner`, `native`, or `video` objects are required. | Object | Defines properties of a audio ad. For examples, see [`adUnit.mediaTypes.video`](#adUnit.mediaTypes.video). |

Reviewer: At least one of the `banner`, `native`, `audio` or `video`

> | [`audio`](#adUnit.mediaTypes.audio) | At least one of the `banner`, `native`, or `video` objects are required. | Object | Defines properties of a audio ad. For examples, see [`adUnit.mediaTypes.video`](#adUnit.mediaTypes.video). |

Reviewer 2: ```suggestion
| [`audio`](#adUnit.mediaTypes.audio) | At least one of the `banner`, `native`, `video` or `audio` objects are required. | Object | Defines properties of a audio ad. For examples, see [`adUnit.mediaTypes.audio`](#adUnit.mediaTypes.audio). |
```

Like this, right?

| `adServer.baseAdTagUrl` | required if `adServer.params` is not defined | string | Your AdServer Ad Tag. The targeting params of the winning bid will be appended. |
| `adServer.params` | required if `adServer.baseAdTagUrl` is not defined | object | Querystring parameters that will be used to construct the video ad tag URL. |

<a name="adUnit.mediaTypes.audio"></a>

#### adUnit.mediaTypes.audio

{: .table .table-bordered .table-striped }
| Name | Scope | Type | Description |
|------------------+-------------+------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `context` | Recommended | String | The audio context. Defaults to 'instream'. |

> | `useCacheKey` | Optional | Boolean | Defaults to `false`. Setting this `true` will use cache url defined in global options also for outstream responses. |

Reviewer 2: ```suggestion
| `useCacheKey` | Optional | Boolean | Defaults to `false`. If set to `true`, the cache url defined in the global options will also be used for outstream responses. |
```

The cache key is always used for instream, right?
